### ISSUE TITLE
please publish master to npm to silence customFds warning

### ISSUE THREAD

**Issue by jeffbski** [TooTallNate/node-gyp#587]
There was a merge to master #533 to fix the node-gyp warning 

```
child_process: customFds option is deprecated, use stdio instead.
```

However **this has not yet been published to npm** so all packages using node-gyp continue to get the deprecation warning.

**Could this be tagged and published so we can take advantage of this fix?**

Thanks!

**Comment by TooTallNate** [comment]
Published v1.0.3. Thanks!

**Comment by jeffbski** [comment]
Awesome. Thanks.
